src/MainUtils.cpp

```diff
@@ -288,6 +317,15 @@ void genConstPackObj(const mlir::OwningModuleRef &module,
 #endif
 }
 
+string getTargetOptions() {
+  string targetOptions = "";
+  if (mtriple != "")
+    targetOptions = "--mtriple=" + mtriple;
+  if (mcpu != "")
+    targetOptions += " --mcpu=" + mcpu;
+  return targetOptions;
+}
+
```

Comment:
It turns out that the same triple/cpu option does work for both the O3 and llc, so its great that the same option is passed for both.